### ISSUE TITLE
Validator participation only decreases with time after the last update

### ISSUE THREAD

**Issue by veiodruida** [aptos-labs/aptos-ait2#29]
Hello team, My validator only decrease the participation since the last update, i was 80 and now i become 22%, and decreasing...

4 vCPUs @ 2.4Ghz 
8 GB RAM 
300 GB NVMe SSD
Ubuntu 20.04 (LTS) x64
Located in Germany - Contabo

I'm running using Docker (image tag - testnet_f5d8013b0a1851da8e078394d83130d3adaf7670 )
Node Address - 69d7fb622a8f75c1d3267b3e5c723fb3ea0d07d1d34c9d866984c6cea3f8d834
My logs - https://drive.google.com/file/d/11e0bYs5YHZcERULKnbhdUCFNDLv3Of0E/view?usp=sharing

Thanks in advance, best regards

**Comment by sherry-x** [comment]
we haven't been getting metrics from your node in the past two days, are you sure your node still running?

**Comment by veiodruida** [comment]
@sherry-x Yes its live, and because of that i have these logs... i try restart the validator since the last update several times, and prune volumes too, i don't no what i need to do anymore, with this.

**Comment by veiodruida** [comment]
@sherry-x  
I'm trying a new way to improve my participation. I am now using Haproxy to redirect the network to a new better server (8 cores, better connection in EUA)and I try again to sync the network using this new architecture. But the consensus doesn't increase and i see all the connections are established. What i missing? please help

tcp   LISTEN    0      4096   38.242.135.185:6181          0.0.0.0:*     users:(("haproxy",pid=8851,fd=9))
tcp   ESTAB     0      0      38.242.135.185:6180    128.199.83.81:56660 users:(("haproxy",pid=8851,fd=223))
tcp   ESTAB     0      0      38.242.135.185:6180     95.216.7.245:39456 users:(("haproxy",pid=8851,fd=146))
tcp   ESTAB     0      0      38.242.135.185:6180      3.72.107.43:40386 users:(("haproxy",pid=8851,fd=167))
tcp   ESTAB     0      0      38.242.135.185:6180   95.217.113.104:33666 users:(("haproxy",pid=8851,fd=246))
tcp   ESTAB     0      0      38.242.135.185:40662 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=191))
tcp   ESTAB     0      0      38.242.135.185:40772 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=333))
tcp   ESTAB     0      0      38.242.135.185:40704 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=241))
tcp   ESTAB     0      257    38.242.135.185:40470 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=100))
tcp   ESTAB     0      0      38.242.135.185:6180     161.97.95.75:40150 users:(("haproxy",pid=8851,fd=56))
tcp   ESTAB     0      0      38.242.135.185:6180  185.197.249.118:34188 users:(("haproxy",pid=8851,fd=140))
tcp   ESTAB     0      0      38.242.135.185:6180   103.183.113.10:46600 users:(("haproxy",pid=8851,fd=217))
tcp   ESTAB     0      0      38.242.135.185:40538 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=45))
tcp   ESTAB     0      0      38.242.135.185:6180    88.99.216.174:55116 users:(("haproxy",pid=8851,fd=199))
tcp   ESTAB     0      0      38.242.135.185:6180  185.209.164.103:53772 users:(("haproxy",pid=8851,fd=283))
tcp   ESTAB     0      0      38.242.135.185:40472 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=102))
tcp   ESTAB     0      0      38.242.135.185:40634 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=164))
tcp   ESTAB     0      8690   38.242.135.185:40456 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=79))
tcp   ESTAB     0      0      38.242.135.185:6180   65.108.136.146:33890 users:(("haproxy",pid=8851,fd=99))
tcp   ESTAB     0      13126  38.242.135.185:40652 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=181))
tcp   ESTAB     0      0      38.242.135.185:6180   147.182.201.72:49272 users:(("haproxy",pid=8851,fd=292))
tcp   ESTAB     0      0      38.242.135.185:6180  187.150.103.113:33812 users:(("haproxy",pid=8851,fd=27))
tcp   ESTAB     0      0      38.242.135.185:40710 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=248))
tcp   ESTAB     0      0      38.242.135.185:6180    209.145.62.91:44138 users:(("haproxy",pid=8851,fd=44))
tcp   ESTAB     0      0      38.242.135.185:40490 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=112))
tcp   ESTAB     0      0      38.242.135.185:6180    35.165.24.123:58139 users:(("haproxy",pid=8851,fd=50))
tcp   ESTAB     0      0      38.242.135.185:40464 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=88))
tcp   ESTAB     0      0      38.242.135.185:6180   38.242.235.125:50042 users:(("haproxy",pid=8851,fd=206))
tcp   ESTAB     0      0      38.242.135.185:40590 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=64))
tcp   ESTAB     0      0      38.242.135.185:41138 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=265))
tcp   ESTAB     0      0      38.242.135.185:40572 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=68))
tcp   ESTAB     0      6655   38.242.135.185:40778 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=339))
tcp   ESTAB     0      0      38.242.135.185:40636 144.126.141.112:6180  users:(("haproxy",pid=8851,fd=166))

**Comment by veiodruida** [comment]
I find the solution by myself. My node is now increasing votes finally. But this only happens after the last code update, and migration to a new server with 8 vCPUs with Haproxy redirecting the network...

**Comment by veiodruida** [comment]
Nice
